.github/workflows/release.yml

```
@@ -79,17 +77,13 @@ jobs:
           cache: npm
       - name: Install dependencies
         run: npm ci
-      - name: Execute version target on all affected packages and release them
-        env:
-          GITHUB_TOKEN: ${{ steps.generate_token.outputs.token }}
-        run: npx nx version @code-pushup/cli-source
-      - name: Deploy to NPM
+      - name: Execute version target on all affected packages, release then publish them
```

I guess the step name is no longer accurate, since there is no `version` target anymore:

```suggestion
      - name: Release packages
```

---

I thought of it but it is doing a bit more than release packages.
This task bumps versions, generate the changelog, release and publish.
So if we want to be accurate...